vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -91,13 +91,14 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.3.4",
     "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:v1.4.1",
     # Azure cloud node manager. Owner: nilo19 (Qi Ni)
-    "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.25.15", # for k8s 1.25.x
     "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.25.18", # for k8s 1.25.x
-    "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.26.11", # for k8s 1.26.x
+    "mcr.microsoft.com/oss/kubernetes/azure-cloud-node-manager:v1.25.23", # for k8s 1.25.x
```

Comment:
Is v1.26.11 still used for now? If so, I think that it is better to keep the old one. Same for other versions.

Reply:
I don't know if the strategy has changed.
1. added a new one:1.26.18
2. keep an old one: 1.26.14
3. remove the oldest one: 1.26.11

Reply:
I suppose we have 2 for a major version?

Reply:
It has followed the rule.